inc/namespace.php

```diff
@@ -51,6 +51,12 @@ function bootstrap() {
 		add_action( 'login_header', __NAMESPACE__ . '\\add_login_logo' );
 	}
 
+	if ( ! empty( $config['favicon-icon'] ) ) {
```

Comment:
@shadyvb I couldn't do a check here for `has_site_icon`,  it would present a "Fatal Error" so I went with your suggestions check if the value was not empty.

Reply:
That works, you can short circuit within the filter if needed. The possibly-redundant function calls shouldn't be a big concern.